mtags/src/main/scala-3/scala/meta/internal/pc/PcInlayHintsProvider.scala

```diff
@@ -77,13 +76,24 @@ class PcInlayHintsProvider(
           if params.implicitParameters() =>
         val labelParts = symbols.map(s => List(labelPart(s, s.decodedName)))
         val label =
-          if allImplicit then labelParts.separated("(", ", ", ")")
+          if allImplicit then labelParts.separated("(using ", ", ", ")")
```

Comment:
How would this work with context bounds e.g `def test[T: Ordering](x: T)(using Int) = ???` 

I think this is one test that should be added to ensure that we don't show evidence params and that we also print `using` or `implicit` depending on last implicit parameter list (context bounds are appended to last implicit parameter list if it exists, and it always adds evidences as implicits).

Also when i use this like this:
```scala
def test[T: Ordering](x: T)(using Int) = ???
test(1)
```
I'd expect it to look like this:
```scala
def test[T: Ordering](x: T)(using Int) = ???
test/*[T: Ordering]*/(1)(using Int)
```

and when there are is no implicit param list then:
```scala
def test[T: Ordering](x: T) = ???
test/*[T: Ordering]*/(1)
```